apps/nextra/pages/en/build/smart-contracts/aptos-standards/fungible-asset.mdx

```diff
@@ -1,689 +1,310 @@
-in objects that contain a FungibleStore resource. These store objects can be
-freely created, and FAs can be moved, split, and combined between them easily.
-
-The standard also supports minting new units and burning existing units with
-appropriate controls.
-
-The different objects involved - `Object<Metadata>` and `Object<FungibleStore>`
-objects, and their relationships to accounts are shown in the diagram below:
-
-{
-/* FIXME replace image
-<div style={{ textAlign: "center" }}>
-  <ThemedImage
-    alt="fungible asset architecture"
-    sources={{
-      light: useBaseUrl('/img/docs/fungible-asset.svg'),
-      dark: useBaseUrl('/img/docs/fungible-asset-dark.svg'),
-    }}
-  />
-</div>
-*/
-}
+# Aptos Fungible Asset (FA) Standard
 
-## Difference with Aptos Coin
-
-FA is a broader category than just coins. While fungible coins are just one
-possible use case of FA, it can represent a wider range of fungible items, such
-as in-game assets like gems or rocks, event tickets, and partial ownership of
-real-world assets. FA provides the flexibility for customizable, detailed
-management and offers a new programming model based on objects. For Aptos coin,
-a `Coin` uses a generic, or the `CoinType`, to support distinct typing within
-the Coin framework. For example, `Coin<A>` and `Coin<B>` are two distinct coins,
-if `A != B`. In contrast, FA does not have a generic in struct definition but
-uses the metadata reference to distinguish the type, which will be further
-explained later. Minimally, [Aptos coin](aptos-coin.mdx) should be
-interchangeable with FA. The migration plan is under discussion.
-
-## Structures
-
-### Metadata Object
-
-Metadata objects with unique addresses define the type of the FAs. Even if
-`Metadata` structs of two `Object<Metadata>` are exactly the same, as long as
-their addresses are different, the FAs points to them would be different. In
-short, the address of the metadata object can be used as **unique identifier**
-of the FA type.
-
-```move
-module 0x1::fungible_asset {
-  #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
-  struct Metadata has key {
-    supply: Option<Supply>,
-    /// Name of the fungible metadata, i.e., "USDT".
-    name: String,
-    /// Symbol of the fungible metadata, usually a shorter version of the name.
-    /// For example, Singapore Dollar is SGD.
-    symbol: String,
-    /// Number of decimals used for display purposes.
-    /// For example, if `decimals` equals `2`, a balance of `505` coins should
-    /// be displayed to a user as `5.05` (`505 / 10 ** 2`).
-    decimals: u8,
-  }
-}
-```
+The Aptos Fungible Asset Standard (also known as “Fungible Asset” or “FA”) provides a standard, type-safe way to define fungible assets in the Move ecosystem. It is a modern replacement for the `coin` module that allows for seamless minting, transfer, and customization of fungible assets for any use case. 
```

Comment:
Maybe a small table describing the capabilities of `coin` vs `fungible asset` might be nice here to better explain why someone might use it

Reply:
Updated with the 2nd improvement (that the stores are created automatically on transfer), but ultimately I think the two standards are supposed to be interchangeable for the other parts, so I don't think a table would help explain that.